src/gameplay/game.ts

```diff
@@ -5,22 +5,25 @@ import _ from 'lodash';
 import Room from './room';
 import usernamesIndexes from '../myFunctions/usernamesIndexes';
 import User from '../models/user';
-// import GameRecord from '../models/gameRecord';
-import DailyGameRecord from '../models/dailyGameRecord';
+import GameRecord from '../models/gameRecord';
+import GameRatingRecord from '../models/gameRatingRecord';
```

Comment:
Can we call this `RatingPeriodGameRecord`? It makes it more clear that it's intended to be a temporary piece of data that is specific to the rating periods. 

Or maybe you have a better suggestion for a name? :)

Reply:
Yeah, I can update it